unit-tests/live/metadata/test-depth-unit.py

```diff
@@ -8,7 +8,7 @@
 
 #############################################################################################
 # get metadata depth units value and make sure it's non zero and equal to the depth sensor matching option value
-test.start("checking depth units on metadata")
+test.start("checking sensor timestanp on metadata")
```

Comment:
Typo
Also looks like this did not need to change. Please check again.

Reply:
Done